basiclti/basiclti-tool/src/java/org/sakaiproject/blti/tool/LTIAdminTool.java

```diff
@@ -1657,8 +1657,8 @@ public void doSingleContentItemResponse(RunData data, Context context) {
 		}
 
 		// Retrieve the tool associated with the content item
-		Long toolKey = foorm.getLongNull(data.getParameters().getString(LTIService.LTI_TOOL_ID));
-		if (toolKey == 0 || toolKey < 0) {
+		Long toolKey = foorm.getLong(data.getParameters().getString(LTIService.LTI_TOOL_ID));
```

Comment:
i believe this change is going to return -1 when passed a null instead of a null